pkg/api/types.go

```diff
@@ -313,6 +313,8 @@ const (
 type PersistentVolumeStatus struct {
 	// Phase indicates if a volume is available, bound to a claim, or released by a claim
 	Phase PersistentVolumePhase `json:"phase,omitempty"`
+	// Current service state of volume
+	Conditions []PersistentVolumeCondition `json:"conditions,omitempty" patchStrategy:"merge" patchMergeKey:"type"`
```

Comment:
What is the semantic use of Conditions elsewhere?  I've seen the types.  I'm curious about their behavior.  Pod's Ready condition seems like it was always be present whereas I thought I saw others that were more temporary markers.

I see the Bound state as a permanent condition.  

The Provisioned condition is arguably permanent or in the list only when requiring external provisioning.  I think this is known at PV creation time (the need for provisioning or not) and can probably be correctly expressed at the birth of the object.

Recycled condition would be the same as Provisioned.  I was planning on using the presence of the condition as the trigger for recycling.  When the volume is released, the controller puts the Recycled(status=false) Condition on the volume. It could be that the condition is always there but status=true as the default and only the change to False triggers it.

Is there a standard behavior I should follow?